client/internal/dns/upstream_test.go

```diff
@@ -49,15 +49,15 @@ func TestUpstreamResolver_ServeDNS(t *testing.T) {
 			timeout:             upstreamTimeout,
 			responseShouldBeNil: true,
 		},
-		//{
```

Comment:
If these scenarios are not valid anymore then please remove from the code. The git will remember for it.

Reply:
I didn't even notice, goland just fixed the syntax but I agree. I will remove it